testrunners/tests/tests.py

```diff
@@ -9,7 +9,7 @@
 import databases
 from libs.differs import jiwer_differ
 from testrunners.tests import TranscriptTest
-
+from apptests import TestModel
```

Comment:
The changes from the tests.py file should be rolled back, as DummyTest can be used.